### PR TITLE
Created news-post.html, section 3

### PR DESCRIPTION
**Pull request by vicwinsj**
- [ ] Created news-post.html
- [ ] Pasted section 3

### CODE REVIEW

File: pages/news-post.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+          <p class="text-lg my-2">The families of the children are also included
+          in the food program and receive monthly distributions of food and
+          hygiene products.</p>
+          <p class="text-lg my-2">We are constantly working to improve the living
+          conditions of the children, and we use much of the surplus we have to
+          repair houses and homes.</p>
+        </p>
+      </div>
+
+    <!-- Newsletter -->
```
Comment (EliNygard) [inline pages/news-post.html:188]: Should Newsletter be a separate <section>? When I look at the design file it looks like they could be two different sections.  Now Newsletter and Support us is placed inside the same section. If this is intentional it's fine :)

File: pages/news-post.html [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+          class="search-drawer_close text-sm pr-5"
+        >
+          Close
+        </button>
+        <!-- JS intention: On click, add -translate-y-full to section id="search-drawer-from-header" and remove translate-y-0 -->
+      </div>
+    </section>
+    <!-- Section 3 -->
+    <!-- Support us -->
+<section class="flex flex-col gap-20">
```
Comment (EliNygard) [inline pages/news-post.html:168]: If Support us and Newsletter will be in separate sections, the Support us sections need a h1-h6 or to be changed to a <div> or another element that does not require a h-element inside it.

**Review by EliNygard — COMMENTED** [review]
I have a question about the use of section. Let me know what you think :)

File: pages/news-post.html (hunk 1 above)
Comment (vicwinsj) [reply pages/news-post.html:244]: I just looked at the updated design file, that was different from the preview picture in the issue. So I intentionally just added the newsletter div into the same section. There haven't been opened any issues for a fourth section of the page containing the newsletter content, so I think it's alright to have it in the same section. But I can ofc update with two different sections.

Comment (benjaminskarding) [reply pages/news-post.html:244]: Yo vic, i see Anne has split the newsletter into a separate section in the updated agency theme dev file, so lets just do that.

Section 4 = newsletter

Comment (EliNygard) [reply pages/news-post.html:244]: Great! I'll wait on that change :)

Comment (EliNygard) [reply pages/news-post.html]: Perfect! Thank you!

**Review by EliNygard — APPROVED** [review]
Perfect! Thank you :)

**Pull request merged**